src/applications/personalization/rated-disabilities/components/RatedDisabilityListItem.jsx

```diff
@@ -17,11 +17,13 @@ const RatedDisabilityListItem = ({ ratedDisability }) => {
             {name}
           </p>
         </div>
-        <div className="vads-l-col--3 medium-screen:vads-l-col--3 small-desktop-screen:vads-l-col--3">
-          <p className="vads-u-font-weight--bold vads-u-font-size--lg vads-u-text-align--right vads-u-margin--0">
-            {ratingPercentage}%
-          </p>
-        </div>
+        {ratingPercentage ? (
+          <div className="vads-l-col--3 medium-screen:vads-l-col--3 small-desktop-screen:vads-l-col--3">
```

Comment:
I don't think you need the `medium-screen:` or `small-desktop-screen:` classes here since they are the same as the default `vads-l-col--3`